### ISSUE TITLE
Mimicking Urination in PKsim/MoBi

### ISSUE THREAD

**Issue by Tpa3** [Open-Systems-Pharmacology/Forum#275]
Hi everyone, 
First off thanks for making such great pieces of software that I use almost every day.

I have a few questions regarding Urine concentrations of compounds in PKsim.

I have built a model for a compound and verified the results against plasma data. However, I was also given measured urine concentrations of the compound and would like to further verify the model against this data. 

From what I can tell the Urine compartment holds the total amount of the removed compound from the blood in PKsim with a static volume and the compound is never removed from the urine compartment. The total amount would work great most of the time, unfortunately this won’t work for my dataset. 

I was wondering if there was a way to simulate the removal of the compound from the compartment to mimic urination in PKsim or MoBi. 

If I can simply remove a percentage of the compound from the compartment at specified times after administration this would suffice to get a rough estimate. Is this something I could do in the Events Building block in MoBi? I could then presumably re-write the Urine compartment’s formula for volume as increasing a function with time that resets after urination occurs. 

Thank you!

TlDr: 
Is there a way to simulate the sudden removal of a compound from the Urine compartment in PKsim or MoBi to mimic urination? If I could remove a certain percentage of the compound from the compartment at specified times I would be a very happy camper.

**Comment by sfrechen** [comment]
Yes, I would follow the proposal of Henrik: adjust your data so that they can be compared with the default PK-Sim observer (concentration in urine, which is actually amount due to the volume of 1 liter).

If you really want to model "emptying of the bladder", you would need to go to MoBi and create respective events at wished time points.

**Comment by sfrechen** [comment]
Well, in that case you need to go to MoBi anyway.
Hence, it would probably more elegant (and easier) to define an event to empty the urine compartment at the desired time point.

**Comment by Tpa3** [comment]
@sfrechen  Thanks for the tip, that is definitely a far more elegant solution. The events BB is the one BB that I haven't totally understood yet, but this will be a wonderful opportunity to do so. I don't know when it changed but it looks like the help documentation for MoBi is a lot more detailed than before, so I'll take a look at that.

Thank you!
